software/wmla120_ansible/validate_firewall_setup.yml

```diff
@@ -1,15 +1,15 @@
 ---
 #If a firewall is enabled, the following default ports must be granted access on all management hosts for IBM Spectrum Conductor Deep Learning Impact: 9243, 9280, 5000, 5001, 27017, and 6379. If you change these ports after installation, make sure to update firewall rules accordingly.
 #
+#
+- name: Include vars of envs_spectrum_conductor_dli_template.yml into the 'ports' variable
+  include_vars:
+    file: envs_spectrum_conductor_dli_template.yml
+    name: ports
 
-- name: Setting Variables
-  set_fact:
-    port_9243: '9243/tcp'
-    port_9080: '9080/tcp'
-    port_5000: '5000/tcp'
-    port_5001: '5001/tcp'
-    port_27017: '27017/tcp'
-    port_6379: '6379/tcp'
+- name: DEBUG - List imported Variables
+  debug:
+    msg: "{{ ports }}"
 
 - name: List all host firewall data
   shell: "firewall-cmd --list-all"
```

Comment:
We need to check if the firewall is running.  This should be changed to "firewall-cmd --state" which returns will return an error (stderr) of 'not running' when not running.

Reply:
Have updated, also have done unit test to verify playbook will not fail on errors. added in new submission.